src/content/resources/whats-new.mdx

```diff
@@ -0,0 +1,55 @@
+---
+
+Keeping track of the latest major changes in Tiptap can be challenging. This page serves as a comprehensive overview over all important changes made to our most important products.
+
+## Tiptap Editor 3.0.0
+
+With the release of Tiptap Editor 3.0.0, we release our third major version of Tiptap Editor. This release includes a list of new features, improvements and bug fixes.
+
+### Getting started with our upgrade guide
+
+Do you already use Tiptap Editor 2.x and want to upgrade to Tiptap Editor 3.x? We've created a comprehensive upgrade guide to help you with the transition.
+
+You can find the upgrade guide [here](/guides/upgrade-tiptap-v2). The guide will guide you through all breaking changes, new features and other updates made to our editor packages that are important for you to know about.
+
+### Breaking Changes
+
+- **Removed UMD builds** - UMD builds have been removed. We recommend using ESM builds instead. UMD builds are not compatible with the new `@tiptap/extension-*` packages.
+- **Migration from `tippy.js` to `floating-ui`** - We replaced the outdated `tippy.js` library with the new `floating-ui` library. This change improves the performance and reliability of our menus and tooltips.
+- **Changes to our Text Style API** - The `@tiptap/extension-text-style` package has seen API updates providing a new `TextStyleKit` extension that encapsulates all possible text styling functionalities in one extension. Default options were also updated.
+- **Commands were changed** - The behavior or names of commands were changed. `clearContent` and `setContent` now emit updates correctly while `setContent` also received a change to it's parameters. `insertContent` was changed to prevent unwanted text splitting.
+- **NodeView `getPos` now can return undefined** - `nodeView.getPos()` can now return `undefined` making a check for `undefined` necessary to handle this case properly.
+- **`editor.getCharacterCount()` was removed** - The `editor.getCharacterCount()` method was removed. It was marked as deprecated in the previous version and is now removed. You can use `editor.storage.characterCount.characters()` instead while using the `CharacterCount` extension.
+- **`considerAnyAsEmpty` option removed from placeholder extension** - The `considerAnyAsEmpty` option was removed from the placeholder extension as it was deprecated already and wasn't respected in the previous version.
+- **Stricter Typing** - 3.0.0 brings stricter typing to the editor. This change improves the overall type safety of the editor and makes it easier to work with but might require some adjustments in your code. We recommend using TypeScript to benefit from the improved type safety.
+
+### New features
+
+- **New `@tiptap/extensions` extension** - The new `@tiptap/extensions` package includes and combines multiple utility extensions. Read more [here](/guides/upgrade-tiptap-v2#extensions-package).
```

Comment:
Instead of "here" being the linked text, I suggest the link could be `[Read more](/guides/upgrade-tiptap-v2#extensions-package)`. This way the clickable area is larger, and [the link text is more accessible](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context).

Reply:
https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context#:~:text=News%20article%20summaries,of%20the%20link.